mx_nuget.package/libgit2/LibGit2Sharp.dll.config

```diff
@@ -0,0 +1,4 @@
+﻿<configuration>
```

Comment:
This file can go too

Reply:
Removed teh file from mx_nuget.package added the file ref in nuspec